### PR TITLE
Renamed pyweed package and added compatibility for django 2.0

### PR DESCRIPTION
**Pull request by insantr**
1. Rename pyweed to pyseaweed based on https://github.com/utek/pyseaweed/commit/9ede653eb6c39d7797ce2d10e9c74b52330ad712
2. Added compatibility with Django >=1.8

### CODE REVIEW

**Comment by khorolets** [comment]
@insantr Hi! Thanks for your work. I suggest to increase version as well.

File: djweed/db_fields.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
 from django.contrib.contenttypes.models import ContentType
-from django.core.urlresolvers import reverse
+try:
+    from django.core.urlresolvers import reverse
+except ImportError:  # Django 1.8
```
Comment (khorolets) [inline djweed/db_fields.py]: I've checked with Django 1.11.1 where I actually use django-weed and there is `reverse` in `django.core.urlresolvers`. So according to notification on this page https://docs.djangoproject.com/en/1.11/ref/urlresolvers/ this change is for Django 2.0 compatibility, not for 1.8, until 2.0 it is working (I guess I want the comment to be more correct)

**Review by khorolets — APPROVED** [review]
Great, thanks!

**Comment by frol** [comment]
@insantr @khorolets  Thank you! I will publish the new version to PyPi by tomorrow morning.

**Comment by frol** [comment]
Done: https://pypi.org/project/django-weed/0.2.3/

/cc @waipbmtd

**Pull request merged**